portfolio/src/test/java/com/google/sps/PortfolioTests.java

```diff
@@ -0,0 +1,45 @@
+package com.google.sps;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import comments.Comment;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+@RunWith(JUnit4.class)
+public final class PortfolioTests {
+
+  @Test
+  public void testCommentInDs() {
+    String comment = "This is a test";
+    long datePosted = 10;
+
+    Entity commentEntity = new Entity("Test");
+    commentEntity.setProperty("text", comment);
+    commentEntity.setProperty("datePosted", datePosted);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(commentEntity);
+
+    Query query = new Query("Test");
+    PreparedQuery results = datastore.prepare(query);
+
+    for (Entity entity : results.asIterable()) {
+      long id = entity.getKey().getId();
+      String text = (String) entity.getProperty("text");
+      long newDatePosted = (long) entity.getProperty("datePosted");
+
+      Assert.assertEquals(text, comment);
+      Assert.assertEquals(newDatePosted, datePosted);
+      datastore.delete(entity.getKey());
+    }
```

Comment:
A test should only perform an assertion, in case of lists we should compare the returned list with an expected one, avoiding the assertion inside of a loop.

I believe that in this case we are expecting that a single result is returned. So I suggest split this test in 3. 

`testThatASingleEntityIsReturned` - where we can just test that the list size is 1.
`testExpectedCommentIsReturned` - where we test that the value for the first list element is the expected Comment.
`testExpectedDatePostedIsReturned` - where we test that the value for the first list element is the expected date.

For a good test case, we should also add negative scenarios, in this case I would suggest query a comment that doesn't exist and confirm that the return value is empty.